admin/class-yoast-form.php

```diff
@@ -327,6 +327,7 @@ public function checkbox_list( $variable, $labels, $attr = [] ) {
 	 * @param bool   $reverse Reverse order of buttons (default true).
 	 * @param string $help    Inline Help that will be printed out before the toggle.
 	 * @param bool   $strong  Whether the visual label is displayed in strong text. Default is false.
+	 *                        Starting from Yoast SEO 16.5, the visual label is forced to bold via CSS.
```

Comment:
If it's always bold now, why would we want to keep this flag? It feels like we can turn off the boldness of the label. Maybe we can't delete it, but can we deprecate it instead?

Also, your code comment & my remark should be applied to the `Light_Switch_Presenter` that has the same option. Does it even make sense to keep the class `switch-light-visual-label__strong` and to add it to the switch?

Reply:
Yep, unsure how to handle the deprecation. Will look at it.